services/github/github-checks-status.service.js

```diff
@@ -1,8 +1,21 @@
 import Joi from 'joi'
-import { pathParams } from '../index.js'
+import { pathParam } from '../index.js'
 import { isBuildStatus, renderBuildStatusBadge } from '../build-status.js'
 import { GithubAuthV3Service } from './github-auth-service.js'
-import { documentation, httpErrorsFor } from './github-helpers.js'
+import {
+  documentation as commonDocumentation,
+  httpErrorsFor,
+} from './github-helpers.js'
+
+const description = `
+Displays the status of a (tag|commit|branch), as reported by the Commit Status API.
```

Comment:
I had included a `(tag|commit|branch)` regex-style format as I assumed we may want to make the description dynamic depending on the example. Having a static one is fine by me, but I'd suggest writing it as `tag, commit, or branch`, or something like that to have a proper English sentence. :)